CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
+## 2.4.0
+
+* Add the ability to bypass CORS preflight requests. This matches the JS implementation
```

Comment:
FWIW open source version of grpc-web does not have any of this code as far as I can see.

Reply:
That's weird, I no longer see it in the public documentation. Either way, this feature is still broadly useful, but I've removed the references to JS gRPC-web parity. I'll speak to the JS gRPC-web folks in the meantime.

Reply:
It's actually my fault - the code is there, I was just searching for it in a wrong way. Most of the functionality comes from Closure Library (https://github.com/google/closure-library/blob/master/closure/goog/net/rpc/httpcors.js#L15) and it is invoked here (https://github.com/grpc/grpc-web/blob/master/javascript/net/grpc/web/grpcwebclientbase.js#L339-L349)

Reply:
Interesting. I don't see it anywhere but I remember the gRPC-web folks recommending Envoy to use with this configuration, but I'm not sure exactly how it works externally.